packages/main/src/MenuItem.ts

```diff
@@ -166,6 +176,22 @@ class MenuItem extends ListItem implements IMenuItem {
 	@property({ type: Boolean, noAttribute: true })
 	_siblingsWithIcon = false;
 
+	/**
+	 * Defines the component selection mode.
+	 * @default "None"
+	 * @private
+	 */
+	@property()
+	_itemSelectionMode: `${ItemSelectionMode}` = ItemSelectionMode.None;
```

Comment:
selection mode for consistency with all other components that provide selection

Reply:
**MenuItem** extends **ListItem** (where `selectionMode` already exists, but with different behavior, so mixing is not possible here.